packages/react-components/src/components/EndCallButton.tsx

```diff
@@ -63,71 +74,128 @@ export const EndCallButton = (props: EndCallButtonProps): JSX.Element => {
     styles ?? {}
   );
 
+  /* @conditional-compile-remove(end-call-options) */
+  const defaultMenuProps: IContextualMenuProps = {
+    items: [
+      {
+        key: 'endForSelf',
+        text: 'Leave',
```

Comment:
These strings will probably be localized later but I'll point it out here just in case.